NGitLab.Mock.Tests/LabelsMockTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Linq;
+using NGitLab.Models;
+using NUnit.Framework;
+
+namespace NGitLab.Mock.Tests
+{
+    public class LabelsMockTests
+    {
+        [Test]
+        public void Test_labels_from_project_can_be_found()
```

Comment:
note for later: we should really find a way to run the same tests for the server and the mock. This would increase the confidence in the mocks

Reply:
Indeed, it would be awesome!